### ISSUE TITLE
Moss is using undefined type from ghistogram

### ISSUE THREAD

**Issue by emad-elsaid** [couchbase/moss#57]
I was trying to use `moss` as a store for `bleve` package, but I get the following error:
```
../../go/src/search/vendor/github.com/couchbase/moss/api.go:173:15: undefined: ghistogram.Histograms
./../go/src/search/vendor/github.com/couchbase/moss/collection.go:98:13: undefined: ghistogram.Histograms
```

where `~/go/src/search` is my client package that vendored `bleve` and `moss`
I see in `modd/api.go:173` that it's using a structu from ghistogram package called `Historgrams` in plural, but this struct doesn't exist in ghistograms package.

is there something I'm missing or that was a typo, or ghistogram was changed overtime and became incompatible with moss latest version?

**Comment by jredl-va** [comment]
@emad-elsaid I too encountered this issue this morning. It turns out dep had vendored the release 0.0.0 of https://github.com/couchbase/ghistogram.

If you change your vendored version to brach master you can build this project correctly.

**Comment by emad-elsaid** [comment]
thanks @jredl-va I will try that, but it caught my attention that this issue didn't have any comments for 23 days now, is the project still maintained?

**Comment by jredl-va** [comment]
@emad-elsaid I don't know. I'm running into this issue for the same reason you are. I'm spiking out usage of https://github.com/blevesearch/bleve and I was unable to build my project. 

I ended up pinning my revision of ghistogram with dep:

```
[[override]]
  name = "github.com/couchbase/ghistogram"
  revision = "d910dd063dd68fb4d2a1ba344440f834ebb4ef62"
```

**Comment by abhinavdangeti** [comment]
Hey @jredl-va @emad-elsaid, Apologies on not getting back sooner. Yes the project is still maintained. With the latest commit on github.com/couchbase/ghistograms - you should be able to see the type Histograms. I believe you'd have to upgrade your revision of the ghistogram package to the latest available which is: d910dd063dd68fb4d2a1ba344440f834ebb4ef62.

We'll also update the vendor'ed version of moss in blevesearch/bleve.
